Packages/MIES/MIES_SweepFormula.ipf

```diff
@@ -4040,6 +4044,27 @@ static Function/WAVE SF_OperationRange(variable jsonId, string jsonPath, string
 	return SFH_GetOutputForExecutorSingle(range, graph, SF_OP_RANGE, dataType = SF_DATATYPE_RANGE)
 End
 
+/// concat(array0, array1, array2, ...)
+static Function/WAVE SF_OperationConcat(variable jsonId, string jsonPath, string graph)
+
+	variable numArgs, i, err, majorType
+
+	numArgs = SFH_CheckArgumentCount(jsonId, jsonPath, SF_OP_CONCAT, 1)
+
+	WAVE result = SFH_GetArgumentAsWave(jsonId, jsonpath, graph, SF_OP_CONCAT, 0, copy = 1, singleResult = 1)
+	Note/K result
+	majorType = WaveType(result, 1)
+
+	for(i = 1; i < numArgs; i += 1)
+		WAVE slice = SFH_GetArgumentAsWave(jsonId, jsonpath, graph, SF_OP_CONCAT, i, singleResult = 1, expectedMajorType = majorType)
+
+		Concatenate/FREE/NP {slice}, result; err = GetRTError(1)
```

Comment:
This does not work correctly if slice is a wave reference wave. See e.g. SF_OperationAnaFuncParamIterate

Reply:
I'm not following: `concat(dataset([1]))` does work. And we also only request a single result.

Reply:
And `concat(dataset([1], [2]))` should IMHO not work.

Reply:
After a chat we agreed nothing needs to be done.